spec/controllers/categories_controller_spec.rb

```diff
@@ -1,5 +1,27 @@
 require 'spec_helper'
 
+# describe CategoriesController, "/create" do
```

Comment:
Thanks!. I added all necessary changes- 
1. deleted all the commented out tests 
2. renamed this feature's name so it will make more sense.